WEB_SERVICE/WEB/_HARDENING/WEB_COOKIE_MISSING_SECURITY_FLAGS/meta.json

```diff
@@ -8,5 +8,10 @@
   "title": "Cookie missing security attributes",
   "cvss_v3_vector": "CVSS:3.0/AV:A/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
   "privacy_issue": true,
-  "security_issue": true
+  "security_issue": true,
```

Comment:
The categories must match the naming of the entries in the Database. These are all wrong.